README.md

```diff
@@ -4,7 +4,7 @@
 [![JetBrains incubator project](https://jb.gg/badges/incubator.svg)](https://github.com/JetBrains#jetbrains-on-github)
 [![Kotlin](https://img.shields.io/badge/kotlin-2.1-blue.svg?logo=kotlin)](http://kotlinlang.org)
 [![CI status](https://img.shields.io/github/checks-status/JetBrains/koog/main)](https://github.com/JetBrains/koog/actions?query=branch%3Amain)
-[![GitHub license](https://img.shields.io/github/license/JetBrains/koog)](LICENSE)
```

Comment:
It's just `LICENSE`, not `LICENSE.txt`

Reply:
 Why is there no link here?

Reply:
What do you mean?

Reply:
That's a link to a file in our repo, to this one: https://github.com/JetBrains/koog/blob/develop/LICENSE

Reply:
Okay, so what worked must be `License.txt`. Because I renamed it because there is no other way to create the license directory

Reply:
Oh, my bad, sorry: I missed the file renaming. Yep, all good here.